hooks/src/index.js

```diff
@@ -140,7 +140,9 @@ export function useImperativeHandle(ref, createHandle, args) {
 	const state = getHookState(currentIndex++);
 	if (argsChanged(state._args, args)) {
 		state._args = args;
-		ref.current = createHandle();
+			if (ref) {
```

Comment:
Really, really small nit 😄 I believe there's 3x tabs in here? And should it be just 2x?